pint/models/phase_jump.py

```diff
@@ -9,11 +9,14 @@
 
 
 class PhaseJump(TimingModel):
-    """This is a class to implement phase jumps 
+    """This is a class to implement phase jumps
     """
     def __init__(self):
         super(PhaseJump, self).__init__()
-        self.add_param(p.maskParameter(name = 'JUMP'))
+        # NOTE: it is dangours to put jump unit as second.
+        # Since JUMP is not always in the unit of time. 
```

Comment:
Ok, I will change it very soon. Is this a 'L1' (before binary ) delay or 'L2' (after binary ) delay?